cfgov/unprocessed/apps/paying-for-college/webpack-config.js

```diff
@@ -36,17 +36,24 @@ const COMMON_MODULE_CONFIG = {
     /* The below regex will capture all node modules
        that start with `cf-` or `cfpb-`.
        Regex test: https://regex101.com/r/zizz3V/5 */
-    exclude: /node_modules\/(?:cf\-.+|cfpb\-.+)/,
+    exclude : [
+        /node_modules\/(?:cf\-.+|cfpb\-.+)/,
+        /\bcore-js\b/,
+        /\bwebpack\/buildin\b/
+      ],
     use: {
       loader: 'babel-loader?cacheDirectory=true',
       options: {
         presets: [ [ '@babel/preset-env', {
+          corejs: 3,
           configPath: __dirname,
           /* Use useBuiltIns: 'usage' and set `debug: true` to see what
              scripts require polyfilling. */
-          useBuiltIns: false,
-          debug: false
-        } ] ]
+          useBuiltIns: 'usage',
```

Comment:
Usage can balloon the JS quite a bit. Check your before/after size. If you set these two lines to:

```
useBuiltIns: 'usage',
debug: true,
```
And run the build you will see what polyfills are required. It may be easier/more performant to refactor the code its referring to polyfilling.

Reply:
I'm definitely concerned about the increased size of the JS, but the need for polyfills is pretty extensive so this seemed like the quickest way to get the app working in IE11. I'm hoping the polyfills can be addressed in some upcoming refactoring and then we can remove this setting.

Reply:
Makes sense. Another way you might consider doing it that we use elsewhere is to include the specific things you need in the modules where they are used, e.g. like [this](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/js/routes/on-demand/mortgage-performance-trends.js#L14-L16) or [this](https://github.com/cfpb/cfgov-refresh/blob/e94774b137503fc4c2d45e13ba44cb1cd13bc71c/cfgov/unprocessed/apps/ccdb-landing-map/js/index.js#L1-L4), that way it's a bit easier to see if and why the core-js polyfills are needed in the future.

Reply:
@anselmbradford Updated to direct imports in https://github.com/cfpb/cfgov-refresh/pull/5869/commits/ba5521766152212dfd811d1d3903ec839df5cb64 -- thanks for the suggestion!

Reply:
Awesome. You might put a comment above the imports saying they're there for IE11